config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
 
+  get 'statistics/count_metric' => 'statistics#count_metric'
```

Comment:
If you decide to modify the method name, don't forget about modifying here as well :)